### ISSUE TITLE
I can't find 'fpcore' module

### ISSUE THREAD

**Issue by tom832** [keiserlab/e3fp-paper#4]
Hello, how can I get python module 'fpcore' in your pipeline.py?
`from fpcore.fconvert import string2ascii, ascii2string`

**Comment by mjke** [comment]
@tom832 that import's from a closed source package that we happened to use in the e3fp paper. sorry for the confusion.

@sethaxen I think these two functions are self-contained enough that we might be able to migrate/add them to this repo. do you still have access to fpcore in seaware and can you run it by me what code would be necessary to release this as a standalone utility script to support `pipeline.py`?

**Comment by sethaxen** [comment]
@mjke it seems all that would need to be ported are the C functions `fconvert_string2ascii` and `fconvert_ascii2string`, which respectively convert from a binary string to Daylight ASCII or vice versa. I don't recall much about what Daylight's ASCII format is, but I wonder if this functionality is now in rdkit.

**Comment by mjke** [comment]
great, I'd forgotten that part was C. I don't think rdkit supports this. would you have the bandwidth to take this on these days?

**Comment by sethaxen** [comment]
> would you have the bandwidth to take this on these days?

Not at the moment, due to vacation. Perhaps in the new year?

I'd be more inclined though to search around for any existing open-source package that provides this functionality. Adding dependencies here isn't an issue, since this is paper code.

**Comment by sethaxen** [comment]
Also, what would we need to do license-wise to port the code here?

**Comment by mjke** [comment]
> I'd be more inclined though to search around for any existing open-source package that provides this functionality. Adding dependencies here isn't an issue, since this is paper code.

yes I was just thinking the same. but @sethaxen what's the real point of `pipeline.py`? on my quick review, it seems to mostly be a collection of utility functions to integrate e3fp with the (not publicly available) seaware package. so it's included here for completeness (and internal-project continuity) but on the public-facing side, mostly to illustrate a usage example?

if so, I don't see how revising it would help @tom832 , since the resulting ascii-encoded "native" fingerprints would only be useful to the seaware package. so perhaps the correct resolution here is to improve the README text and the .py file header comments to clarify this, instead? right now, the README could make it seem `pipeline.py` is the repo landing point, whereas [/project/scripts/](/keiserlab/e3fp-paper/tree/master/project/scripts) is the real point, right?

**Comment by tom832** [comment]
> > I'd be more inclined though to search around for any existing open-source package that provides this functionality. Adding dependencies here isn't an issue, since this is paper code.
> 
> yes I was just thinking the same. but @sethaxen what's the real point of `pipeline.py`? on my quick review, it seems to mostly be a collection of utility functions to integrate e3fp with the (not publicly available) seaware package. so it's included here for completeness (and internal-project continuity) but on the public-facing side, mostly to illustrate a usage example?
> 
> if so, I don't see how revising it would help @tom832 , since the resulting ascii-encoded "native" fingerprints would only be useful to the seaware package. so perhaps the correct resolution here is to improve the README text and the .py file header comments to clarify this, instead? right now, the README could make it seem `pipeline.py` is the repo landing point, whereas [/project/scripts/](/keiserlab/e3fp-paper/tree/master/project/scripts) is the real point, right?

Thanks for relying, I haven't install seaware yet. And this issue is not that serious. Anyway, I can use e3fp to generate fps as expected.
